jwt/index.ts

```diff
@@ -0,0 +1,54 @@
+import config from 'config';
+import memoize from 'memoizee';
+import axios, { AxiosPromise } from 'axios';
+import ms from 'ms';
+import jwt from 'jsonwebtoken';
+import urljoin from 'url-join';
+
+export enum JwtVerificationStrategy {
+  API = 'API',
+  Signature = 'SIGNATURE',
+  None = 'NONE',
+}
+
+interface VerifyJWT {
+  (token, options?): Promise<boolean>;
+}
+
+export const dummyVerifyJWT: VerifyJWT = (
+  token,
+  { resolveWith } = { resolveWith: true },
+) => Promise.resolve(resolveWith);
+
+export const verifyJWTByApi: VerifyJWT = token =>
+  axios
+    .post(urljoin(config.egoApiRoot, '/introspect'), {
```

Comment:
Does this endpoint exist? or is it planned?

Reply:
does not exist yet, but is planned